src/Microsoft.Health.Fhir.Api/Registration/FhirServerApplicationBuilderExtensions.cs

```diff
@@ -23,10 +24,21 @@ public static class FhirServerApplicationBuilderExtensions
         /// <param name="app">The application builder instance.</param>
         /// <returns>THe application builder instance.</returns>
         public static IApplicationBuilder UseFhirServer(this IApplicationBuilder app)
+        {
+            return app.UseFhirServer(null);
```

Comment:
Nit: Should this method have an EnsureArg for app like the other UseFhirServer method?